lib/AuroraLip/Common/FormatDictionary_List.cs

```diff
@@ -248,6 +248,10 @@ public static partial class FormatDictionary
             new FormatInfo(".vol", "RTDP", FormatType.Archive, "Arc Rise Archive", "Imageepoch"),
             new FormatInfo(".wtm", "WTMD", FormatType.Texture, "Arc Rise Texture", "Imageepoch"),
 
+            //Natsume
+            new FormatInfo(".pBin", "pBin", FormatType.Archive, "Harvest Moon Archive", "Natsume"),
+            new FormatInfo(".tex", FormatType.Texture, "Harvest Moon Texture", "Natsume"){ Class = typeof(FIPAFTEX), IsMatch = FIPAFTEX.Matcher },
```

Comment:
I often have issues with this "FormatDictionary_List.cs" file.  For instance, I swear I started with that line you have listed and nothing was read.  I assumed it was because something about how it is searching was assuming the magic was a 4 character class.  I didn't really look into it.

The

`{ Class = typeof(XXX), IsMatch = XXX.Matcher }` is my fallback for when this file just fails to cooperate with me.

Reply:
Just tried making this change.  No files get outputted like that.